pkgs/by-name/li/lib60870/package.nix

```diff
@@ -0,0 +1,30 @@
+{ cmake
+, lib
+, stdenv
+, fetchFromGitHub
+}:
+stdenv.mkDerivation {
```

Comment:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Reply:
Thanks. Addressed in 1a4591f71467ef88dedd2a7f7ba246abd185f1dd